language/cloud-client/README.md

```diff
@@ -0,0 +1,30 @@
+# Getting Started with Google Cloud Natural Language API and the Google Cloud Client libraries
+
+[Google Cloud Natural Language API][language] provides natural language
+understanding technologies to developers, including sentiment analysis, entity
+recognition, and syntax analysis. This API is part of the larger Cloud Machine
+Learning API.
+
+These sample Java applications demonstrate how to access the Cloud Natural
+Language API using the [Google Cloud Client Library for Java][google-cloud-java].
+
+[language]: https://cloud.google.com/natural-language/docs/
+[google-cloud-java]: https://github.com/GoogleCloudPlatform/google-cloud-java
+
+## Quickstart
+
+Install [Maven](http://maven.apache.org/).
+
+Build your project with:
+
+	mvn clean package -DskipTests
+
+You can then run a given `ClassName` via:
+
+	mvn exec:java -Dexec.mainClass=com.example.language.ClassName \
+	    -DpropertyName=propertyValue \
+		-Dexec.args="any arguments to the app"
```

Comment:
I'd do:

    -Dexec.args="arg1 'arg 2' arg3"

to make clear that spaces in a single arg have to be quoted.